cluster/olm/clusterserviceversion.yaml.tmpl

```diff
@@ -59,3 +200,55 @@ spec:
       url: https://twitter.com/upbound_io
     - name: Slack
       url: https://slack.crossplane.io/
+  # TODO(muvaf): Consider copying this from README automatically, preserving the
```

Comment:
There isn't an easy way to do this that preserves the indentation for a long multi-line string.